backend/queue-service/src/main/kotlin/com/ticketqueue/queue/scheduler/BatchApproveScheduler.kt

```
@@ -16,28 +18,37 @@ import java.util.UUID
 
-        for (rawId in scheduleIds) {
-            val scheduleId = try {
-                UUID.fromString(rawId)
-            } catch (e: IllegalArgumentException) {
-                logger.warn { "Invalid scheduleId format in active-schedules, skipping: $rawId" }
-                continue
-            }
+            for (rawId in scheduleIds) {
+                val scheduleId = try {
+                    UUID.fromString(rawId)
+                } catch (e: IllegalArgumentException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)